releasenotes/2021-February-3.md

```diff
@@ -0,0 +1,192 @@
+
+# Breaking API Changes
+1. New flow for collection subscriptions:
+```js
+    // Subscribe to participants currently in a Call
+    call.remoteParticipants.forEach(p => {
+        subscribeToRemoteParticipant(p);
+    })
+
+    call.on('remoteParticipantsUpdated', e => {
+        // Subscribe to new participants in the call
+        e.added.forEach( p=> {
+            subscribeToRemoteParticipant(p)
+        })
+        e.removed.forEach( p => {
+            unsubscribeToRemoteParticipant(p)
+        })
+    });
+
+    function subscribeToParticipant(p) {
+        // Subscribe to participant's current video/screensharing streams
+        p.videoStreams.forEach(v => { handleRemoteStream(v) });
+        // Subscribe to particiapnt's new video/screensharing streams
+        p.on('videoStreamsUpdated', e => {
+            e.added.forEach(v => { handleRemoteStream(v) })
+        })
+    }
+}
+```
+2. Changed Call.callerIdentity variable to Call.callerInfo which is now of type CallerInfo
+```js
+export declare interface CallerInfo {
+    /**
+    * Identity of the caller
+    */
+    identity: CommunicationUserIdentifier | PhoneNumberIdentifier | CallingApplicationIdentifier | MicrosoftTeamsUserIdentifier | UnknownIdentifier | undefined;
+}
+```
+
+4. Changed Call.isIncoming: boolean to Call.direction: CallDirection
+```js
+    export declare type CallDirection = 'Incoming' | 'Outgoing';
+```
+
+3. Type name changes from @azure/communication-common
+```js
+    CommunicationUserCredential -> CommunicationTokenCredential
+    CommunicationUser -> CommunicationUserIdentifier
+    PhoneNumber -> PhoneNumberIdentifier
+    CallingApplication -> CallingApplicationIdentifier
+```
+
+# New features
+
+## Call recording management
+
+Call recording is an extended feature of the core `Call` API. You first need to obtain the recording feature API object:
+
+```js
+const callRecordingApi = call.api(Features.Recording);
+```
+
+Then, to can check if the call is being recorded, inspect the `isRecordingActive` property of `callRecordingApi`, it returns `Boolean`.
+
+```js
+const isResordingActive = callRecordingApi.isRecordingActive;
+```
+
+You can also subscribe to recording changes:
+
+```js
+const isRecordingActiveChangedHandler = () => {
+  console.log(callRecordingApi.isRecordingActive);
+};
+
+callRecordingApi.on('isRecordingActiveChanged', isRecordingActiveChangedHandler);
+```
+
+## Call Transfer management
+
+Call transfer is an extended feature of the core `Call` API. You first need to obtain the transfer feature API object:
+
+```js
+const callTransferApi = call.api(Features.Transfer);
+```
+
+Call transfer involves three parties *transferor*, *transferee*, and *transfer target*. Transfer flow is working as following:
+
+1. There is already a connected call between *transferor* and *transferee*
+2. *transferor* decide to transfer the call (*transferee* -> *transfer target*)
+3. *transferor* call `transfer` API
```

Comment:
nit, Transferror should put current call on hold first, before invoking 'transfer' API